vunit/test/bench.py

```diff
@@ -106,19 +106,18 @@ def _check_architectures(design_unit):
                     )
                 )
 
-    def create_tests(self, simulator_if, elaborate_only, test_list=None):
+    def create_tests(self, simulator_if, elaborate_only, test_list=None, precompiled=None):
```

Comment:
Is `precompiled` something to be set per test? I'd say it's something related to the tool/backend, which should be unrelated to the tests.

Reply:
This was sort of a wack-a-mole implmentation, I just wanted to make the precompiled param available for the sim_if classes. So I let it trickle down the same path as elaborate_only.

I did that as I'm not that familiar with the inner workings of this code base. Could I do it in another way?
Perhaps I could add a set-function for precompiled on the sim_if function, and perhaps at the same time add one for the elaborate_only param?

Reply:
Or add it to the configuration object? since it seems to handle sim_options

Reply:
Updated the constructor for the sim_if classes to accept both elaboration and precompiled.

Since both of these won't change during a VUnit session.